core/app/views/layouts/refinery/admin.html.erb

```diff
@@ -2,22 +2,30 @@
 <%= render '/refinery/html_tag' %>
   <% content_for :meta, tag(:meta, name: 'refinerycms', content: Refinery.version) %>
   <%= render 'refinery/admin/head' %>
-  <body class="<%= action_name %> <%= I18n.locale %>">
+  <body class="<%= action_name %> <%= I18n.locale %> cms-interface">
     <%= render 'refinery/admin/menu' %>
     <%= render '/refinery/site_bar' %>
     <div id='tooltip_container'></div>
-    <div id="admin_container" class="clearfix">
-      <div id="page_container">
-        <div id="page">
-          <div id="content" class="clearfix">
-            <div id="flash_container">
-              <%= render '/refinery/no_script' %>
-              <%= render '/refinery/message' %>
+    <section id="wrapper" class="dimmer">
+      <% if content_for?(:admin_content) %>
```

Comment:
Basically, what's the case for this new feature and how is it tested? :smile: Where is this used? Is this documented somewhere?

This also appears to remove the standard `yield` - does overriding `admin_content` mean you have to do that yourself?